src/coreclr/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.xml

```diff
@@ -6,5 +6,8 @@
     <type fullname="Internal.Runtime.InteropServices.ComponentActivator" feature="System.Runtime.InteropServices.EnableConsumingManagedCodeFromNativeHosting" featurevalue="false">
       <method signature="System.Boolean get_IsSupported()" body="stub" value="false" />
     </type>
+    <type fullname="System.Reflection.Metadata.MetadataUpdater" feature="System.Reflection.Metadata.MetadataUpdater.IsSupported" featurevalue="false">
```

Comment:
It's a bit unfortunate this is duplicated in the ILLink.Substitutions.xml files between coreclr and mono.

Since this is a public class/property, they won't differ between mono and coreclr, so can you reduce the duplication and put this entry in [https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Shared.xml](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Shared.xml) instead?